roles/lead_data_engineer.md

```diff
@@ -4,13 +4,13 @@ UK-based in Birmingham, Bristol, Glasgow, Manchester, Newcastle, London & Swanse
 
 Please apply for this role at [www.madetech.com/careers](https://www.madetech.com/careers).
 
-Our Lead Data Engineers enable public sector organisations to embrace a data-driven approach by providing data platforms and services that are high-quality, cost-efficient, and tailored to the client's needs. They lead teams to develop, operate, and maintain these services, as well as ensuring they provide maximum value to data consumers, including analysts, scientists, and business stakeholders.
+Our Lead Data Engineers enable public sector organisations to deliver and run data platforms and services that are high-quality, cost-efficient, and tailored to the client's needs. They support teams to develop, operate, and maintain these services, as well as ensuring they provide maximum value to data consumers, including analysts, scientists, and business stakeholders.
 
 ## What does the job entail?
 
-At Made Tech we want to positively impact the future of the country by using technology to improve society, for everyone. We want to empower the public sector to deliver and continuously improve digital services that are user-centric, data-driven and freed from legacy technology. A key component of this is developing modern data systems and platforms that drive informed decision-making for our clients. You will also work closely with clients to help shape their data strategy.
+At Made Tech we want to positively impact the future of the country by using technology to improve society, for everyone. We want to empower the public sector to deliver and continuously improve digital services that are secure, user-centric, data-driven and freed from legacy technology. A key component of this is developing modern data systems and platforms that drive informed decision-making for our clients. You will also work closely with clients to help shape their data strategy.
 
-As a Lead Data Engineer, you may play one or more roles according to our clients' needs. Sometimes the role is very hands-on and you'll serve as a technical lead for a project, focusing on both delivering engineering work as well as upskilling members of the client team. At other points, you might play more of a technical architect role, ensuring that the wider context of a delivery is considered and maintaining alignment between the operational and analytical aspects of the engineering solution.
+You may play one or more roles according to our clients' needs. Sometimes the role is very hands-on and you'll contribute to a project, focusing on both delivering engineering work as well as upskilling members of the client team. At other points, you might play more of a technical architect role, ensuring that the wider context of a delivery is considered and maintaining alignment between the operational and analytical aspects of the engineering solution.
```

Comment:
"serve as a technical example to project members"?

Reply:
I've gone with "Sometimes the role is very hands-on and you'll both manage the team and contribute to a project, delivering engineering work including code as well as upskilling members of the client team."